### PR TITLE
Set language on App created

### LINKED ISSUE #691: Chinese language settings are not saved if the user closes the wallet
I change the language settings in the wallet to Chinese, then collapse/close it. After a while, I open the wallet and see that my language settings have not been saved. The default is English.

Запис екрана 2020-12-01 о 4 46 17 пп

This issue is reproduced on the development branch.
Mac OS 11.0.1
Superhero 3832d26 / v.0.4.2
Superhero ca07c27 / v.0.4.2

### PR DESCRIPTION
**Pull request by CedrikNikita**
fixes #691

### CODE REVIEW

**Review by etharner — CHANGES_REQUESTED** [review]

File: src/store/actions.js [hunk 1]
```diff
@@ -68,6 +68,11 @@ export default {
     ).address;
   },
 
+  async setCurrentLanguage({ commit }, language) {
+    commit('setCurrentLanguage', language);
+    fetchAndSetLocale(language);
```
[the remark this answers is not in the document]
Comment (CedrikNikita) [reply src/store/actions.js]: Actually i agree with that.

File: src/store/plugins/languages.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    name: 'English',
+    getMessages: () => import(/* webpackChunkName: "locale-en" */ '../../popup/locales/en.json'),
+  },
+  cn: {
+    name: '中文',
+    getMessages: () => import(/* webpackChunkName: "locale-cn" */ '../../popup/locales/cn.json'),
+  },
+};
+
+export const fetchAndSetLocale = async (languageCode) => {
```
Comment (davidyuk) [inline src/store/plugins/languages.js:26]:
```suggestion
const fetchAndSetLocale = async (languageCode) => {
```

File: src/store/plugins/languages.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  if (!i18n.availableLocales.includes(languageCode)) {
+    const messages = (await languages[languageCode].getMessages()).default;
+    i18n.setLocaleMessage(languageCode, messages);
+  }
+  i18n.locale = languageCode;
+};
+
+export const preferredLocale = (() => {
```
Comment (davidyuk) [inline src/store/plugins/languages.js:34]:
```suggestion
const preferredLocale = (() => {
```

File: src/store/plugins/languages.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  const code = navigator.language.split('-')[0];
+  return languages[code] ? code : fallbackLocale;
+})();
+
+export default (store) => {
+  store.registerModule('languages', {
+    namespaced: true,
+    state: {
+      activeCode: store.state.languages ? store.state.languages.activeCode : preferredLocale,
```
Comment (davidyuk) [inline src/store/plugins/languages.js:43]: Let's write it in a modern way 😄
```suggestion
      activeCode: store.state.languages?.activeCode || preferredLocale,
```

**Review by davidyuk — APPROVED** [review]

**Review by etharner — APPROVED** [review]
Cool, LGTM

**Comment by Liubov-crypto** [comment]
Tested, looks good on my side.

**Pull request merged**